ui/src/views/Denylist/ViewDenylistAddresses/index.jsx

```diff
@@ -0,0 +1,56 @@
+import { hot } from 'react-hot-loader';
+import React, { PureComponent } from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import PlusIcon from 'mdi-react/PlusIcon';
+import Denylist from './Denylist';
+import Dashboard from '../../../components/Dashboard';
+import Search from '../../../components/Search';
+import Button from '../../../components/Button';
+import HelpView from '../../../components/HelpView';
+
+@hot(module)
+@withStyles(theme => ({
+  plusIconSpan: {
+    ...theme.mixins.fab,
+  },
+}))
+export default class ViewDenylist extends PureComponent {
+  state = {
+    searchTerm: '',
+  };
+
+  handleSearchSubmit = searchTerm => {
+    this.setState({ searchTerm });
+  };
+
+  handleCreate = () => {
```

Comment:
What are we creating? Change to `handle<the-thing-that-we-are-creating>Create`.

Reply:
This is to navigate to the Add Address page. I think the name should be `handleAddressAdd` but we have an identically named function in another file that handles the actual mutations to add address. Is it okay if we use the same name here or should go for something different?

Reply:
Feel free to use the same name.